text/0000-slack.md

```diff
@@ -0,0 +1,59 @@
+# Meta
+[meta]: #meta
+- Name: Migrate to CNCF Slack
+- Start Date: 2021-12-07
+- Author(s): [@samj1912](https://github.com/samj1912)
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes that we move our independent slack instance to the CNCF slack instance.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+CNCF slack is an umbrella slack enterprise instance that quite a few CNCF projects use. As Buildpacks is maturing as a CNCF project, using the CNCF slack will put us closer to our sibling projects and their community. The CNCF slack also comes with CNCF support around spam prevention, moderation and integration with LFX insights. This will help us manage and track the growth of our community. CNCF slack is also an enterprise instance which means that we will have unlimited retention of messages and the ability to create people groups to mention maintainer teams. We will also be able to use private channels if needed.
+
+As a part of CNCF slack onboarding, we can also migrate all of our data and channels onto the CNCF workspace, preserving any past history of conversations that took place in the Buildpacks slack. For more details please see [CNCF - Migrating workspaces](https://slack.com/intl/en-gb/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another)
+
+# What it is
+[what-it-is]: #what-it-is
+
+This RFC proposes that we move the Buildpacks slack instance from an independent instance to the CNCF instance.
+
+# How it Works
```

Comment:
I'd like to see some more detail of the general channel migration mapping. Do we have a sense of how that will look? Will everything just be prefaced with `#buidlpacks-`?

Reply:
We might also take this opportunity to prune some inactive channels. These ones seem most used:

* arm
* buildpack-authors (maybe instead of buildpacks-buildpack-authors this could buildpacks-authors?)
* distribution
* general
* implementation (maybe rename this to lifecycle?)
* kpack
* learning-team
* pack-cli
* platform
* spec
* tekton

Reply:
To help move this along, what do we think of these mappings?

* arm -> buildpacks-arch-arm
* buildpacks-authors -> buildplacks-authors-team
* general -> buildpacks
* implementataion -> buildpacks-implementation-team
* <del>kpack</del>
* learning -> buildpacks-learning-team
* platform -> buildpacks-platform-team
* pack-cli -> buildpacks-platform-pack
* spec -> buildpacks-spec
* <del>tekton</del> - can't bring this over

Reply:
Also announcements, ci, registry, maintainers, mentoring, windows and distribution.

Reply:
edited the original post.

Reply:
`buildpacks-platform-pack` doesn't feel very discoverable. why not `buildpacks-pack-cli`?

Reply:
@jkutner I was trying to prefix platforms with "platform", but happy to do what people think is most successful.

Reply:
We decided to keep it to pack-cli for now since the CNCF slack allows us to easily rename channels. We can always rename later if we prefer to keep things consistent across platforms.